src/main/java/tutoraid/logic/commands/AddStudentToLessonCommand.java

```diff
@@ -80,9 +69,8 @@ public CommandResult execute(Model model) throws CommandException {
         lessonToAddToStudent.addStudent(studentToAddToLesson);
         studentToAddToLesson.addLesson(lessonToAddToStudent);
 
-        model.updateFilteredStudentList(Model.PREDICATE_SHOW_ALL_STUDENTS);
-        model.updateFilteredLessonList(Model.PREDICATE_SHOW_ALL_LESSONS);
-        model.viewList(HIGH);
```

Comment:
Might be more useful to keep this so that the tutor can add one student after another without having to type `list` each time

Reply:
I see, I will try switching back to showing all the students and lessons